src/templates/CentralTemplateProvider.ts

```diff
@@ -126,8 +141,14 @@ export class CentralTemplateProvider {
         }
 
         if (result) {
-            return result;
+            return {
+                functionTemplates: result.functionTemplates.filter(f => provider.includeTemplate(f)),
+                bindingTemplates: result.bindingTemplates.filter(b => provider.includeTemplate(b))
+            };
         } else if (latestTemplatesError !== undefined) {
+            if (!context.errorHandling.suppressDisplay) {
+                ext.outputChannel.show();
```

Comment:
Because there's actually several errors that have occurred and were printed to the output - getting cached, latest, _and_ backup all must've failed if we got to this point.

Also this should hardly ever happen, so I don't mind being a little annoying and popping up the output channel